README.md

```diff
@@ -5,6 +5,7 @@ https://omg.network/ for that. ⚠️**
 
```

Comment:
While we are here, let's remove the above as it is a dead link which can be exploited.